### PR TITLE
extend datepicket to 1.0

### PR DESCRIPTION
**Pull request by itkg-nanne**

### CODE REVIEW

File: BackofficeBundle/Form/Type/DatePickerType.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+
+/**
+ * Class DatePickerType
+ */
+class DatePickerType extends DateType
+{
+    /**
+     * @param FormView $view
```
Comment (itkg-alavieille) [inline BackofficeBundle/Form/Type/DatePickerType.php]: align comment

**Comment by itkg-alavieille** [comment]
else phpdoc :+1:

File: BackofficeBundle/Form/Type/DatePickerType.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+     * @param FormView      $view
+     * @param FormInterface $form
+     * @param array         $options
+     */
+    public function finishView(FormView $view, FormInterface $form, array $options)
+    {
+        parent::finishView($view, $form, $options);
+        $view->vars['format'] = $options['format'];
+    }
+    /**
```
Comment (nicolasThal) [inline BackofficeBundle/Form/Type/DatePickerType.php]: you need to add an empty lines between the methods

File: BackofficeBundle/Resources/views/Form/form_div_layout.html.twig [hunk 3]
```diff
@@ -135,3 +135,17 @@
...
+
+{% block oo_date_picker_widget %}
+    {% spaceless %}
+    {% if form.vars.widget == 'single_text' %}
+    	<div class="input-group">
+        {% set attr = attr|merge({ 'class': (attr.class|default('') ~ ' datepicker')|trim, 'data-dateformat': form.vars.format|default('yyyy-mm-dd')|lower }) %}
+    {% endif %}
+    {{ block('date_widget') }}
+    {% if form.vars.widget == 'single_text' %}
+			<span class="input-group-addon"><i class="fa fa-calendar"></i></span>
```
Comment (nicolasThal) [inline BackofficeBundle/Resources/views/Form/form_div_layout.html.twig]: could you fix the indentation

**Pull request merged**